python/python/ert/geo/surface.py

```diff
@@ -44,18 +45,27 @@ class Surface(BaseCClass):
     _isqrt        = GeoPrototype("void   geo_surface_isqrt( surface )")
 
     
-    def __init__(self, filename):
+    def __init__(self, filename=None, ny=None, xinc=None, yinc=None,
+                                      xstart=None, ystart=None,
+                                      nx=None, angle=None, internal_z=None):
         """
         This will load a irap surface from file. The surface should
         consist of a header and a set z values.
         """
-        if os.path.isfile( filename ):
-            c_ptr = self._alloc(filename , True)
-            super(Surface , self).__init__(c_ptr)
+        if filename is not None:
+            filename = str(filename)
```

Comment:
The `internal_z` should in 99/100 cases be `True`. At least it should default to `True` - alternatively it could even be hardcoded as `internal_z = True` inside the method.